featurebyte/query_graph/graph.py

```diff
@@ -283,7 +283,7 @@ def load(self, graph: QueryGraphModel) -> Tuple["QueryGraph", Dict[str, str]]:
     def extract_operation_structure(
         self,
         node: Node,
-        keep_all_source_columns: bool,
+        keep_all_source_columns: bool = True,
```

Comment:
This addresses the comment in https://github.com/featurebyte/featurebyte/pull/1524#discussion_r1263359589.